lib/videoroom/room.ex

```diff
@@ -49,6 +50,8 @@ defmodule Videoroom.Room do
         :error -> nil
       end
 
+    simulcast? = Application.fetch_env!(:membrane_videoroom_demo, :simulcast?)
```

Comment:
As discussed let's move this on the client side and change query param into checkbox